console/src/main/webapp/WEB-INF/spring/webmvc-config.xml

```diff
@@ -72,13 +72,27 @@
   </bean>
 
   <!-- LDAP connection -->
-  <bean id="contextSource" class="org.springframework.ldap.core.support.LdapContextSource">
+  <bean id="singleContextSource" class="org.springframework.ldap.core.support.LdapContextSource">
     <property name="url" value="${ldapScheme}://${ldapHost}:${ldapPort}"/>
     <property name="base" value="${ldapBaseDn}"/>
     <property name="userDn" value="${ldapAdminDn}"/>
     <property name="password" value="${ldapAdminPassword}"/>
   </bean>
 
+  <bean id="ldapContextValidator" class="org.springframework.ldap.pool.validation.DefaultDirContextValidator">
+  </bean>
+  
+  <bean id="contextSource" class="org.springframework.ldap.pool.factory.PoolingContextSource" destroy-method="destroy">
+    <property name="contextSource" ref="singleContextSource"/>
+    <property name="dirContextValidator" ref="ldapContextValidator"/>
+    <property name="testOnBorrow" value="${ldap.pool.testOnBorrow:true}"/>
+    <property name="maxActive" value="${ldap.pool.maxActive:8}"/>
+    <property name="minIdle" value="${ldap.pool.minIdle:0}"/>
```

Comment:
Shouldn't we always have at least one idle connection by default ?

Reply:
I don't know, should we? to what end?

Reply:
I'm not sure to fully understand how this works, but I'm tempted to think that having always at least one connection available in the pool will improve fetch time.

Reply:
and reduce slapd log spam..

Reply:
We can have an idle connection always ready, yet note we didn't before (e.g. no connection pool at all). I'll change it so the default is one by your request though.

Reply:
done. No `connection_read(19): no connection!` in the ldap logs.